docker-compose.yml

```diff
@@ -58,6 +59,9 @@ services:
         CMAKE_VERSION: "${CMAKE_VERSION:-3.20.2}"
         CCACHE_VERSION: "${CCACHE_VERSION:-4.1}"
         BASE_IMAGE: nvidia/cudagl:${CUDA_VERSION:-11.0}-devel-${LINUX_VERSION:-ubuntu18.04}
+    environment:
+      <<: *common_environment_variables
+      DOCKER_WORKDIR: "${PWD}"
```

Comment:
any reason you chose not to include `DOCKER_WORKDIR` in `common_environment_variables`?

Reply:
I didn't want to set it for the `runtime` containers